gitpod-io/gitpod
# [docker-up] Remove slirp4netns

## Description
This PR removes the remnants of slirp4netns from docker-up. Now that workspaces run their own network namespace, there is no reason to maintain this code.

## How to test
`docker run --rm -it -p 8080:8080 alpine:latest`
expect the container to start and the port to be detected as active.

## Release Notes

```release-note
NONE
```
